server/data/data-jpa/src/main/resources/META-INF/persistence.xml

```diff
@@ -26,6 +26,12 @@
     <persistence-unit name="James" transaction-type="JTA">
        <provider>org.apache.openjpa.persistence.PersistenceProviderImpl</provider>
         <jta-data-source>osgi:service/javax.sql.DataSource/(osgi.jndi.service.name=jdbc/james)</jta-data-source>
+        <class>org.apache.james.mailbox.jpa.mail.model.JPAMailbox</class>
```

Comment:
Nope, I would prefer having these enriched JPA persistence file in your Guice project